### PR TITLE
Breakable tiles

### PR DESCRIPTION
**Pull request by CodePokeX**
Breakable tilesBreakable tiles for tank to pass through added for tank to pass through added

### CODE REVIEW

**Comment by ady-1811** [comment]
Is there no breakable_tile.gd?

**Comment by CodePokeX** [comment]
There should be

**Comment by yukitya-1811** [comment]
Well, you might have forgotten to git add it. Check once. type `git status` and see if it's in untracked files

**Comment by ady-1811** [comment]
It's not in the scripts folder. Move it there and then commit and push again

File: BreakableTiles.gd [hunk 1]
```diff
@@ -0,0 +1,22 @@
+extends StaticBody2D
+class_name BreakableTile
+var count:int =0
+@export var tilemap: TileMap
+
+func _ready():
+	$Sprite2D.hide()
+
+func break_tile():
+	print(count)
```
Comment (ady-1811) [inline BreakableTiles.gd]: Remove this print statement or comment it out

File: BreakableTiles.gd [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	count+=1
+	if count==1:
+		$Sprite2D.show()
+	#print(global_position)
+	var pos=tilemap.local_to_map(global_position)
+	#print(pos)
+	#pos.x=pos.x+2
+	#pos.y=pos.y+1
+	#tilemap.erase_cell(0,pos)
```
Comment (ady-1811) [inline BreakableTiles.gd]: Remove redundant code, unless it's required for debug purposes. If it is required for debug then comment on how it's important

File: BreakableTiles.gd [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+func _ready():
+	$Sprite2D.hide()
+
+func break_tile():
+	print(count)
+	count+=1
+	if count==1:
+		$Sprite2D.show()
+	#print(global_position)
```
Comment (ady-1811) [inline BreakableTiles.gd]: Same as previous comments, either delete or explain use in debugging

File: BreakableTiles.gd [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+@export var tilemap: TileMap
+
+func _ready():
+	$Sprite2D.hide()
+
+func break_tile():
+	print(count)
+	count+=1
+	if count==1:
+		$Sprite2D.show()
```
Comment (ady-1811) [inline BreakableTiles.gd]: Try and rename this Sprite2D to CrackedTile instead of just keeping it as Sprite2D

**Comment by CodePokeX** [comment]
Can I do it by tomorrow morning?

**Review by ady-1811 — CHANGES_REQUESTED** [review]
Check these out

**Review by ady-1811 — APPROVED** [review]

**Pull request merged**